charms/kserve-controller/requirements-integration.in

```diff
@@ -1,6 +1,6 @@
 aiohttp
 jinja2
-juju==3.0.4
+juju<3.0
```

Comment:
Please add a comment that states why we are pinning to this version.

Reply:
done